### ISSUE TITLE
+ataraxy fails kerodon tests

### ISSUE THREAD

**Issue by zakak** [duct-framework/duct#67]
Hello and thanks for all your open source work, including duct.

I noticed `lein new duct demo +ataraxy +example` creates tests that fail, though `http://localhost/example` loads fine in the browser.

The kerodon `(status? 200)` test is looking for `[:response :status]`:

```clojure
;; with +ataraxy +example
(deftest smoke-test    
  (testing "example page exists"               
    (-> (session handler)                      
        (visit "/example") ;; {:response [:ataraxy.response/ok "This is an example handler"] ...}
        (has (status? 200) "page exists")))) ;; fails
```

```clojure
;; with +example
(deftest smoke-test    
  (testing "example page exists"               
    (-> (session handler)                      
        (visit "/example")  ;; {:response {:status 200, :headers {"Content-Type" "text/html; charset=utf-8"}, :body "This is an example handler"}
        (has (status? 200) "page exists")))) ;; passes
```

**Comment by weavejester** [comment]
Thanks for opening the issue. Just wanted to say that I have actually noticed this, and that I'm thinking about how best to solve it.

**Comment by weavejester** [comment]
Should be fixed in 0.10.4.

**Comment by zakak** [comment]
Works for me now, thanks!
